Reply/app/src/main/res/layout/activity_main.xml

```diff
@@ -22,20 +22,22 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         app:layoutFullscreen="@{true}"
+        backgroundSurfaceElevation="@{@dimen/m3_sys_elevation_level5}"
```

Comment:
This needs the `app:` prefix

Reply:
This is cool but I think you can remove it since you're setting the window background to level5 in MainActivity.kt